File: nahomy-c2-backend/src/domain/entities/FuncionPrealmacenada.ts
export class FuncionPrealmacenada {
  constructor(
    private readonly _id: number,
    private _nombreFuncion: string,
    private _comandoSistema: string,
  ) {}

  get id(): number {
    return this._id
  }

  get nombreFuncion(): string {
    return this._nombreFuncion
  }

  get comandoSistema(): string {
    return this._comandoSistema
  }

  actualizarComando(comando: string): void {
    this._comandoSistema = comando
  }
}

File: packages/server/src/domain/entities/funcion_prealmacenada.py
class FuncionPrealmacenada:
    def __init__(self, id: int, nombre_funcion: str, comando_sistema: str):
        self._id = id
        self._nombre_funcion = nombre_funcion
        self._comando_sistema = comando_sistema

    @property
    def id(self) -> int:
        return self._id

    @property
    def nombre_funcion(self) -> str:
        return self._nombre_funcion

    @property
    def comando_sistema(self) -> str:
        return self._comando_sistema

    def actualizar_comando(self, comando: str) -> None:
        self._comando_sistema = comando
